events/git-and-github.md

```diff
@@ -0,0 +1,126 @@
+# Git & Github Workshop
+
+## Aim of the Workshop
+
+At the end of the workshop, make sure that audience are comfortable with the following
+
+- Why Version Control Systems are need?
+- Usage of basic Git commands to track their projects - Adding and Committing
+- Be able to push and pull code from Github
+- Be able to send a PR to an Open Source Repository
+
+## Before the workshop
+
+Share an instructions file like [this](https://github.com/kossiitkgp/git-and-github-workshop-2020/blob/main/installation.md), along with the poster of the workshop. The instructions should contain
+
+- How to install Git for Windows, Linux, Mac seperately
+- For Windows users, make that they install `git bash`
+- How to verify that git is installed
+- Instructions on how to create a Github Account
+
+In the post mention the fact that we will be available for installation issues before 15 minutes of the workshop. Help with the installation issues before the workshop begins.
+
+## Flow of the workshop
+
+-1. **How will the workshop proceed**
+
+   Explain that Git and Github are important parts of Software Development. Tell them how the workshop will proceed like when should audience ask queries. Tell the audience that doing hands-on by themselves along with the workshop will help them learn better. Encourage them to ask questions, as some of the Git concepts are non-intuitive.
+
+0. **Terminal Commands**
+
+   It might be a good idea to introduce basic linux Commands, as they will be used through out the workshop. You don't have to cover this topic in detail. Just important commands like - `ls`, `cat`, `echo` , `cd`, `mkdir`, `pwd` , `rm`would suffice. Windows users can be asked to try these commands on Git Bash. 
+
+1. **Why do we need something like Git**
```

Comment:
I guess the introduction of GitHub is missing. the audience must have a basic idea of why to use GitHub, its motive and the basic difference between foss, proprietary software..., so we can add like - Why do we need something like GitHub..